README.md

```diff
@@ -53,8 +55,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+O(n). The parser has to check each elements and pair ends together, which is O*n/2), but then, ignoring ignoring the factor of division, it would be O(n)
```

Comment:
It _is_ O(n), but I think you're doing the analysis for speed, and not memory usage...

# -1